### DISCUSSION TITLE
ComfyUI UI-update. Mouse behavior changed? I can't.

### DISCUSSION THREAD

**Discussion by bkmikaelo7** [comfyanonymous/ComfyUI#4448] [Q&A]
Hello. Did you change the mouse middle button behaviour for the latest UI update? The mouse wheel now scrolls instead of zooming. I can't do this. Cannot find a setting to revert to old behaviour of middle wheel zooming. Please add one if possible.

**Comment by ltdrdata** [comment]
Please post any requests related to the front end here.
https://github.com/Comfy-Org/ComfyUI_frontend/issues
